src/Core/Services/GraphQLSchemaCreator.cs

```diff
@@ -161,21 +163,24 @@ public ISchemaBuilder InitializeSchemaAndResolvers(ISchemaBuilder schemaBuilder)
         /// <exception cref="DataApiBuilderException"></exception>
         private DocumentNode GenerateSqlGraphQLObjects(RuntimeEntities entities, Dictionary<string, InputObjectTypeDefinitionNode> inputObjects)
         {
+            // Dictionary to store object types for:
+            // 1. Every entity exposed for MySql/PgSql/MsSql/DwSql in the config file.
+            // 2. Directional linking entities to support nested insertions for M:N relationships for MsSql. We generate the directional linking object types
+            // from source -> target and target -> source.
```

Comment:
Here, you intend to convey that for the linking table, two objects and entities will be created right.

For example, Book-Author relationship, LinkingObject_bookAuthor and LinkingObject_authorBook will be created. Likewise, LinkingEntity_bookAuthor and LinkingEntity_authorBook. 

Adding this in the comment, will make it easier to understand

Reply:
No. For a single linking table, only one object is created. If the relationship is defined in both the entities' `relationships` section, then only two objects are created. I have to update the comment. Thanks for noticing!